Sources/Actions/Feed.php

```diff
@@ -2930,9 +2931,9 @@ public static function cdataParse(string $data, string $ns = '', bool $force = f
 					}
 
 					if (substr($data, $pos + 1, 1) == '/') {
-						$cdata .= ']]></' . $ns . ':' . substr($data, $pos + 2, $pos2 - $pos - 1) . '<![CDATA[';
+						$cdata .= ']]></' . $ns . ':' . substr($data, $pos + 2, (int) ($pos2 - $pos - 1)) . '<![CDATA[';
 					} else {
-						$cdata .= ']]><' . $ns . ':' . substr($data, $pos + 1, $pos2 - $pos) . '<![CDATA[';
+						$cdata .= ']]><' . $ns . ':' . substr($data, $pos + 1, (int) ($pos2 - $pos)) . '<![CDATA[';
```

Comment:
Casting the result of the expression to int is unnecessary. The result of the arithmetic operation will already be an integer.